drivers/bridge/bridge.go

```diff
@@ -560,6 +603,9 @@ func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) err
 
 		// Setup DefaultGatewayIPv6
 		{config.DefaultGatewayIPv6 != nil, setupGatewayIPv6},
+
+		// Add intewr-network communication rules
+		{true, setupNetworkIsolationRules},
```

Comment:
it doesn't matter. in any condition, only if iptables is enabled, we should install these rules. 
If the user explicitly disables iptables, for a network (for whatever reason .. maybe they have another script that handles the iptables), then we have to honor that and not program our rules there.